frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -577,7 +577,7 @@ describe("scenarios > question > notebook", () => {
       popover()
         .last()
         .within(() => {
-          cy.findByText("Hour of Day").scrollIntoView();
+          cy.findByText("Hour of day").scrollIntoView();
           // This is an implicit assertion - test fails when there is more than one string when using `findByText` instead of `findAllByText`
           cy.findByText("Minute").click();
```

Comment:
@ariya  I don't understand how this passed for you?
I am running it locally and it fails without exception.

Reply:
![Uploading image.png…]()